faces.py

```diff
@@ -111,11 +111,27 @@ def get_random_person_from_all_batches(cursor, current_user):
                              last_name,
                              image_url
                       FROM people
-                      WHERE person_id != %s
+                      WHERE person_id != %s AND NOT image_url LIKE '%%no_photo_%%'
                       ORDER BY random()
-                      LIMIT 4""",
+                      LIMIT 1""",
                    [current_user])
-    return [x for x in cursor.fetchall()]
+
+    person = cursor.fetchall()
+
+    cursor.execute("""SELECT person_id,
+                             first_name,
+                             middle_name,
+                             last_name,
+                             image_url
```

Comment:
We talked about this in person, and I don't think it makes sense to fix now, but I want to note it here so I/we don't forget later:

For the hints, the only information we care about is the `first_name`; everything else is unused.